File: 6. Python/EX07.py
# Solicita o nome do trabalhador
nom = input("# Nome: ")

# Solicita o valor pago por hora e converte para float
valHor = float(input("# Valor por hora: "))

# Solicita o número de horas trabalhadas e converte para float
horTra = float(input("# Horas trabalhadas: "))

# Calcula o pagamento multiplicando o valor por hora pelas horas trabalhadas
pag = valHor * horTra

# Imprime o resultado formatado com duas casas decimais
print(f"# O pagamento para {nom} deve ser {pag:.2f}")

File: 4. Java/EX07.java
package Apostila;

import java.util.Locale;
import java.util.Scanner;

public class EX07
{
    public static void main(String[] args) 
    {
        // Declara variáveis
        int horTra;
        String nom;
        double pag, valHor;
        
        // Define localização
        Locale.setDefault(Locale.US);
        
        // Cria Scanner
        try (Scanner ent = new Scanner(System.in))
        {
            // Entrada de dados
            System.out.print("# Nome: ");
            nom = ent.nextLine();
            
            System.out.print("# Valor por hora: ");
            valHor = ent.nextDouble();
            
            System.out.print("# Horas trabalhadas: ");
            horTra = ent.nextInt();
        }

        // Calcula o pagamento
        pag = valHor * horTra;
        
        // Exibe o pagamento
        System.out.println("# O pagamento para " + nom + " deve ser " + String.format("%.2f", pag));        
    }
}
